### PR TITLE
[v8.12] 8.12.2 release

### PR DESCRIPTION
**Pull request by Zimmi48**
I will also submit the release notes for 8.12.2 as a PR on `master`.

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -1336,6 +1336,25 @@ Changes in 8.12.1
   fixes `#12332 <https://github.com/coq/coq/issues/12332>`_,
   by Théo Zimmermann and Jim Fehrle).
 
+Changes in 8.12.2
+~~~~~~~~~~~~~~~~~
+
+**Notations**
+
+- **Fixed:**
+  8.12 regression causing notations mentioning a coercion not being used
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:1345]:
```suggestion
  8.12 regression causing notations mentioning a coercion to be ignored
```

or "to not be used"

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -1336,6 +1336,25 @@ Changes in 8.12.1
...
+  (`#13436 <https://github.com/coq/coq/pull/13436>`_,
+  fixes `#13432 <https://github.com/coq/coq/issues/13432>`_,
+  by Hugo Herbelin).
+
+**Tactics**
+
+- **Fixed:**
+  8.12 regression in the inference of implicit arguments in :tacn:`exists`
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:1353]:
```suggestion
  8.12 regression: incomplete inference of implicit arguments in :tacn:`exists`
```

**Review by jfehrle — COMMENTED** [review]
Minor tweaks

**Comment by Zimmi48** [comment]
Thanks for your suggestions. I've also applied them in #13608 (can you take care of merging the latter)?

**Pull request merged**